cmd/arc_summary/Makefile.am

```diff
@@ -1 +1,13 @@
 dist_bin_SCRIPTS = arc_summary.py arc_summary3.py
+
+#
+# The arc_summary.py script is compatibile with both Python 2.6 and 3.4.
+# As such the python 3 shebang can be replaced at install time when
+# targeting a python 2 system.  This allows us to maintain a single
+# version of the source.  The arc_summary3.py script requires Python 3.
+#
+if HAVE_PYTHON_2
```

Comment:
I'd change this flag to `USING_PYTHON_2`. Let's discourage the idea that Python 2 is something we want anyone to default to.

Reply:
Done